src/atlas4py/CMakeLists.txt

```diff
@@ -8,10 +8,12 @@ set(CMAKE_CXX_STANDARD 17)
 include(FetchContent)
 
 function(copy_target ct_tgt)
-    add_custom_target(copy_${ct_tgt} ALL
-        COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:${ct_tgt}> ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
-        DEPENDS ${ct_tgt}
-    )
+    if(NOT TARGET copy_${ct_tgt})
```

Comment:
This change just allows calling `copy_target` multiple times with the same argument.